homeassistant/components/dhcp/__init__.py

```diff
@@ -45,15 +46,23 @@
 FILTER = "udp and (port 67 or 68)"
 REQUESTED_ADDR = "requested_addr"
 MESSAGE_TYPE = "message-type"
-HOSTNAME = "hostname"
-MAC_ADDRESS = "macaddress"
-IP_ADDRESS = "ip"
+HOSTNAME: Final = "hostname"
+MAC_ADDRESS: Final = "macaddress"
+IP_ADDRESS: Final = "ip"
```

Comment:
I would love to be able to rename these `ATTR_HOSTNAME`, `ATTR_MACADDRESS`, `ATTR_IP`
```suggestion
HOSTNAME: Final = "hostname"
MAC_ADDRESS: Final = "macaddress"
IP_ADDRESS: Final = "ip"
```

Would it be OK to add the three ATTR_xxx constants, and mark these three as obsolete somehow?

Reply:
It's easy enough to fix inside `core`, but I guess that would be a breaking change for custom components.

Reply:
I would skip that change for now.

Reply:
I totally understand